src/middlewares/isChannelAdmin.middleware.js

```diff
@@ -0,0 +1,27 @@
+import  Channel  from '../models/channel.models.js';
```

Comment:
Do not use import as type in package.json is not set to module. in all other files we have use require import format.